Indicators/StandardDownsideDeviation.cs

```diff
@@ -0,0 +1,62 @@
+using System;
```

Comment:
nit - missing license header file

Think `StandardDownsideDeviation` should have unit tests of it's own